index.js

```diff
@@ -60,6 +60,23 @@ const wrapWord = (rows, word, columns) => {
 	}
 };
 
+// Trims spaces from a string ignoring invisible sequences
+const stringVisibleTrimSpacesRight = str => {
+	const words = str.split(' ');
+	const first = 0;
+	let last = words.length;
+
+	while (last > first) {
+		if (stringWidth(words[last - 1]) > 0) {
+			break;
+		}
+
+		last--;
+	}
+
+	return words.slice(0, first).join('') + words.slice(first, last).join(' ') + words.slice(last, words.length).join('');
```

Comment:
Please remove `words.slice(0, first).join('')` as it is always an empty string.

Personally I'd also remove the `first` variable entirely and just use `0` where appropriate though I don't feel strongly about this.

Reply:
@stroncium ping

@sindresorhus do you see any other issues with this PR?

Reply:
@coreyfarrell done with a bonus. Sorry for taking so long. I suppose github decided I've read this when I glanced over email. =/ (Particularly annoying feature, if anyone knows how to disable this I'd be happy to listen.)

Reply:
Github includes a link to a 1x1 pixel image like `<img src="https://github.com/beacon/id...">`.  Maybe if you've told your email client that github emails are trusted this bypasses the normal privacy protection?  Every github email shows me the warning `To protect your privacy, Thunderbird has blocked remote content in this message.`

Reply:
Well, we'll see. I'm using Inbox, but it gets shut down today, hopefully gmail got this feature.